monitor/census.go

```diff
@@ -1009,6 +1021,34 @@ func InitCensus(nodeType NodeType, version string) {
 			TagKeys:     baseTags,
 			Aggregation: view.Count(),
 		},
+		{
+			Name:        "ai_whip_transport_bytes_received",
+			Measure:     census.mAIWhipTransportBytesReceived,
+			Description: "Number of bytes received on a WHIP connection",
+			TagKeys:     append([]tag.Key{census.kAIStreamName}, baseTags...),
```

Comment:
Isn't tagging with stream name an issue/leak? I think then the number of lines exposed with `/metrics` will grow with each new stream coming.

Reply:
Yeah I wasn't sure about this, I thought it was ok since we have metrics by streamID in normal studio streaming.
<img width="1769" alt="Screenshot 2025-04-02 at 15 00 53" src="https://github.com/user-attachments/assets/c52a2338-e697-494e-9900-fed338ec9e6d" />

Reply:
Hmm, I think the problem here is that in studio, people reuse stream ID, while in DayDream, we create a stream for each request. So, I'd suggest to not include it. We'll still have it in logs if needed.

Reply:
Good point ok, will remove